migrations/2022-11-21-204256_user-following/up.sql

```diff
@@ -0,0 +1,11 @@
+-- create user follower table with two references to persons
+create table person_follower (
+    id serial primary key,
+    person_id int references person on update cascade on delete cascade not null,
+    follower_id int references person on update cascade on delete cascade not null,
+    published timestamp not null default now(),
+    pending boolean not null,
+    unique (follower_id, person_id)
+);
+
+alter table community_follower alter column pending set not null;
```

Comment:
Now that you've removed the default, you need to select and update any of those columns that might be null. I highly suggest keeping the default instead, pretty sure that won't require the update then.

Reply:
Added the update (already tested with null value). I prefer to clean this stuff up and not allow nonsensical values if possible.